automation/taskcluster/decision_task.py

```diff
@@ -218,9 +240,11 @@ def nightly_to_production_app(is_staging, version_name):
     elif command == 'raptor':
         ordered_groups_of_tasks = raptor(result.staging)
     elif command == 'nightly':
-        nightly_version = datetime.datetime.now().strftime('Nightly %y%m%d %H:%M')
+        now = datetime.datetime.now().strftime('%y%m%d %H:%M')
+        nightly_version = 'Nightly {}'.format(now)
         ordered_groups_of_tasks = release('nightly', 'geckoNightly', result.staging, nightly_version) \
             + nightly_to_production_app(result.staging, nightly_version)
+        ordered_groups_of_tasks += release_as_fennec(result.staging, 'Fenix-as-Fennec Nightly {}'.format(now))
```

Comment:
nit: I wonder if "Nightly" can be confusing here since it is not "Fennec Nightly" but just a nightly build of "Fennec" (the release version). :)

Reply:
Hmm, it's still a nightly build, though I see how it's not clear how it's a "fennec" kind of build.
What about "Signed-as-Fennec Nightly {}"?

Reply:
Yeah, it is still a nightly, just not the "Firefox Nightly" that is on Google Play (with app id org.mozilla.fennec_aurora [for reasons]). "Signed-as-Fennec" is a bit better, yeah!